bin/elixir

```diff
@@ -69,8 +69,17 @@ ERL=""
 
 # Stores erl arguments preserving spaces/quotes (mimics an array)
 erl () {
-  eval "E$E=\$1"
-  E=$(($E + 1))
+  eval "E${E}=\$1"
+  E=$((E + 1))
+}
+
+# Checks if a line starts with char
+# starts_with "$LINE", "$CHAR"
+starts_with () {
+case $1 in
+  "$2"*) true;;
+  *) false;;
+esac
```

Comment:
I think the previous version clearer. Can we please revert this one?

Reply:
I prefer this change since it is more clear to me and easier to contribute to, subjectively speaking.